### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by alokendumazumder** [paganpasta/coss-pytorch#2]
Hi,

I am unable to undersdtand the structure of your saved models. You have provided links, in which it has data.pkl and a data folder with few "no format" files. Also, can you provide a quick and detailed overview on how to replicate the Tab 1?

**Comment by paganpasta** [comment]
Hi @alokendumazumder, 

Once you download the `model_best.pth.tar` file, on most pytorch versions, you should be able to do this:

```python
ckpt = torch.load('model_best.pth.tar', map_location=torch.device('cpu'))
```

Hope this helps. Let me know if the issue persists.

**Comment by alokendumazumder** [comment]
Hi @paganpasta . Thank you for getting back to me so quickly. 

As I am new to the area of knowledge distillation, I have a few queries:

1. The link of the weights, I believe they are students's weights, right? I have loaded the teacher's weights from the SEED repo.
2. To reproduce Table 1, I believe you have trained (distilled from the teacher) student from Imagenet1K (In Imagenet1K there are three training files, task1, task2, and task3, here task1 and task2 are merged as a single trainfile, I guess you have used the task1 and task2 ?). Are the results classified on the val set ?
3. For Table2, you have used the pre-trained weights of students (distilled from students in Tab 1), and performed classification on Cifar and other datasets, right?
